PIL/ImageFont.py

```diff
@@ -233,15 +233,16 @@ def load(filename):
 def truetype(font=None, size=10, index=0, encoding="",
              layout_engine=None):
     """
-    Load a TrueType or OpenType font file, and create a font object.
-    This function loads a font object from the given file, and creates
-    a font object for a font of the given size.
+    Load a TrueType or OpenType font from a file or stream, and create
+    a font object.
```

Comment:
I think we can just change this instance to `file like object`, and leave the next one as file.

Reply:
Nitpick: `file-like object` for both.